docs/conf/cfp/east-2020.rst

```diff
@@ -0,0 +1,40 @@
+Write the Docs East 2020 Call for Venues
+========================================
+
+We're also excited to start planning another US conference in 2020.
+The goal will be to make closer to folks on the east coast,
+but we haven't figured out exactly where to host it yet.
+
+The plan is to host it sometime in the fall,
+so that it is nicely spaced out with our Portland conference.
+
+Requirements
+------------
+
+We have some lose requirements for venues that we're looking for.
```

Comment:
change "lose" to "loose" -- I also think you can eliminate the rest of the sentence after "venues"

We have some loose requirements for venues.